backend/app/core/authz/authorization.py

```diff
@@ -47,6 +96,7 @@ async def _construct_enforcer(model: str) -> AsyncEnforcer:
     def enforce(
         cls,
         action: AuthorizationAction,
+        object_fetcher: type[SubjectResolver] = DefaultResolver,
```

Comment:
We have many parameter with default values. Might be good to make these keyword only, to avoid confusion?

```suggestion
        action: AuthorizationAction,
        *,
        object_fetcher: type[SubjectResolver] = DefaultResolver,
```